variables.tf

```diff
@@ -79,3 +79,62 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "cluster_ingress_hostname" {
+  type        = string
+  description = "Ingress hostname of the IKS cluster."
+  default     = ""
+}
+
+variable "operator_namespace" {
+  type        = string
+  description = "operator namespace"
+  default     = "cpd-operators"
+}
+
+variable "cpd_namespace" {
+  type        = string
+  description = "cpd namespace"
+  default     = "gitops-cp4d-instance"
+}
+
+variable "service_account_name" {
+  type        = string
+  description = "The name of the service account for wkc"
+  default     = "wkc-iis-sa"
+}
+
+variable "sccs" {
```

Comment:
Why is this exposed as a variable? If someone removed the anyuid scc then this would all fail to install, right? We shouldn't be exposing this implementation detail as a variable.

Reply:
implemented the change